src/stories/containers/Finances/utils/types.ts

```diff
@@ -41,3 +41,8 @@ export interface MomentDataItem extends Team {
   totalActuals: number;
   lastModified: DateTime;
 }
+export type Metrics = 'Budget' | 'Actual' | 'Forecast' | 'Net Expenses On-chain' | 'Net Expenses Off-chain';
```

Comment:
This should be in singular as it is all the possible metrics but when it is used it represent only one metric unless it is used as an array

Reply:
you are right, done